src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java

```diff
@@ -384,4 +394,126 @@ private static StringBuilder getLiveCaseLoadReport(ListingData listingData) {
         return sb;
     }
 
+    private static StringBuilder getServedClaimsReport(ListingData listingData) {
```

Comment:
I think all the logic for this report deserves to be in its own class.

Reply:
can we do that as an improvement in another iteration?

Reply:
I mean after releasing this.